modules/zstr/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "homepage": "https://github.com/mateidavid/zstr",
+    "maintainers": [
+        {
+            "email": "bcr-maintainers@bazel.build",
```

Comment:
Feel free to add yourself as a maintainer

Reply:
Done :-)